### DISCUSSION TITLE
追加の質問があります。

### DISCUSSION THREAD

**Discussion by Forseti4** [Yowkees/keyball#405] [Q&A]
マルチメーターで導通テストを行い、PCBの洗浄も試みましたが、特に変化はありませんでした。
現在、'DFG'だけではなく、'ERT'も押されているようです。
前回は'DFG'が入力されていても他のキーも入力されましたが、
現在は'DFGERT'だけが入力され、他の文字を接続しても入力されません。解決策はありませんか？

**Comment by Forseti4** [comment]
今は再び通常に戻りました。'DFG'だけが連続入力され、他のキーは入力されます。しかし、Aを入力するとQ1、Sを入力するとW2が同時に入力されます。全てのホットスワップソケットとダイオードを取り外して再製作すれば解決するかもしれませんか？
![20231031_113036](https://github.com/Yowkees/keyball/assets/146462035/59adabaa-092c-43ab-b278-6b48f91e8aa9)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

部品を除去して再半田付けしても修理できるかもしれませんが、まず下記をご確認ください。

まずProMicroを取り外した上で、下画像の赤点（ROW2）と黄色で示した点との導通をすべて確認してください。
症状から推測すると、どれかがショートしているはずです。（赤点と青点が導通するのはOKです。）

このショートを解消する必要があります。

写真の光の加減かも知れませんが、下画像左端の緑丸で示した部分に半田クズが付着していないでしょうか？
こんな感じで基板上の半田クズが配線に触れてしまっている部分がないか入念にチェックいただければ幸いです。
非常にキレイに半田付けされていますので、原因はそれほどたくさんは無いと思います。おそらく１か所修正できればすぐに動き出すと思いますので、お手数ですがご確認お願いいたします。

![KB61_ROW2fix](https://github.com/Yowkees/keyball/assets/84209772/fe686945-f234-468e-8281-8fb2c39f6281)

**Reply by Forseti4** [reply]
お返事いただきありがとうございます。抵抗値が変わっていたので、確認のためにピンソケットを取り外していたところ、ピンソケットが壊れてしまいました。交換用の部品を注文しましたので、到着次第再確認いたします。ありがとうございます。
